libkbfs/block_tree.go

```diff
@@ -269,6 +271,24 @@ func (bt *blockTree) getNextDirtyBlockAtOffset(ctx context.Context,
 	return ptr, parentBlocks, block, nextBlockStartOff, startOff, nil
 }
 
+// TODO: rename these structs
+// structs used for passing data to and from GetBlocksForOffsetRange tasks
+type task struct {
+	ptr        BlockPointer
+	pblock     *BlockWithPtrs
+	pathPrefix []parentBlockAndChildIndex
+	startOff   Offset
+	endOff     Offset
+	prefixOk   bool
+	getDirect  bool
+}
+type result struct {
+	pathFromRoot    []parentBlockAndChildIndex
+	ptr             *BlockPointer
```

Comment:
Why not use `result.Block != nil` as an indication that there's a block, instead of making this a pointer?